src/templates/toolkit/bet/static.html.tt

```diff
@@ -50,7 +50,7 @@
                         </div>
                         <div class="big-col">
                            <div id="expiry_type_duration">
-                              <input id="duration_amount" type="number" min="1" max="366" class="small_width_input" /><label></label>
```

Comment:
@4p00rv why we remove `min, max`, we use to validate the form. @negar-binary can you confirm this